lib/yaml/__init__.py

```diff
@@ -71,16 +73,49 @@ def compose_all(stream, Loader=Loader):
     finally:
         loader.dispose()
 
-def load(stream, Loader):
+def load(stream=None, Loader=None):
```

Comment:
Why are you making the Loader argument optional again?

Reply:
Was a rebase mess up. Already fixed locally.

Like I said still working on a lot of things with this. I'll ping you when I having something to look at.